pillar.example

```diff
@@ -1,50 +1,20 @@
 filebeat:
   lookup:
-    config_path: /etc/mycustom/filebeat/filebeat.yml
-    config_source: salt://mycustom/filebeat/filebeat.jinja
-    runlevels_install: True
-
```

Comment:
your still merging filebeat:lookup in map.jinja, please provide some example of an override.